api/populateDatabase.php

```diff
@@ -0,0 +1,102 @@
+    curl_close($ch);
+
+    return $result;
+}
+
+$propertyArray = properties($propertyUrl);
+$statusArray = properties($statusUrl);
+$typeArray = properties($typeUrl);
+
+function getArmadilloEstates(): PDO
+{
+    $db = new PDO('mysql:host=DB;dbname=armadilloEstates', 'root', 'password');
+    return $db;
+}
+
+function insertIntoPropertiesTable(PDO $db, array $propertyData): bool
+{
+    $query = $db->prepare("INSERT INTO `properties` (`AgentRef`,
+                                                                 `Address1`,
+                                                                 `Address2`,
+                                                                 `Town`,
+                                                                 `Postcode`, 
+                                                                 `Description`, 
+                                                                 `Beds`, 
+                                                                 `Price`, 
+                                                                 `Image`, 
+                                                                 `PropertyType`, 
+                                                                 `Status`) 
+                                                        VALUES (:AGENT_REF, 
+                                                                :ADDRESS_1, 
+                                                                :ADDRESS_2, 
+                                                                :TOWN, 
+                                                                :POSTCODE, 
+                                                                :DESCRIPTION, 
+                                                                :BEDROOMS, 
+                                                                :PRICE, 
+                                                                :IMAGE, 
+                                                                :TYPE, 
+                                                                :STATUS);");
+    return $query->execute($propertyData);
+}
+
+function insertIntoStatusTable(PDO $db, array $statusData): bool
+{
+    $query = $db->prepare("INSERT INTO `status` (`StatusId`,
+                                                            `StatusName`) 
+                                                        VALUES (:ID, 
+                                                                :STATUS_NAME);");
+    return $query->execute($statusData);
+}
+
+function insertIntoTypesTable(PDO $db, array $typeData): bool
+{
+    $query = $db->prepare("INSERT INTO `types` (`TypeId`,
+                                                            `TypeName`) 
+                                                        VALUES (:ID, 
+                                                                :TYPE_NAME);");
+    return $query->execute($typeData);
+}
+
+
+
+function validateUsrInput(array $newBread) :bool
```

Comment:
Shouldn't have a space before `:`, but after it instead.

Reply:
Removed